test/functional/services/browser.ts

```diff
@@ -109,27 +128,45 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
      * @param {number} yOffset Optional
      * @return {Promise<void>}
      */
-    public async moveMouseTo(element: any, xOffset: number, yOffset: number): Promise<void>;
-    public async moveMouseTo(element: WebElementWrapper): Promise<void>;
+    public async moveMouseTo(element: any, xOffset: number, yOffset: number): Promise<any>;
+    public async moveMouseTo(element: WebElementWrapper): Promise<any>;
     public async moveMouseTo(
       element: WebElementWrapper,
       xOffset?: number,
       yOffset?: number
-    ): Promise<void> {
-      const mouse = (driver.actions() as any).mouse();
-      const actions = (driver as any).actions({ bridge: true });
-      if (element instanceof WebElementWrapper) {
-        await actions
-          .pause(mouse)
-          .move({ origin: element._webElement })
-          .perform();
-      } else if (isNaN(xOffset!) || isNaN(yOffset!) === false) {
-        await actions
-          .pause(mouse)
-          .move({ origin: { x: xOffset, y: yOffset } })
-          .perform();
-      } else {
-        throw new Error('Element or coordinates should be provided');
+    ): Promise<any> {
+      switch (this.browserType) {
+        case Browsers.Firefox: {
+          // Workaround for scrolling bug in Firefox
+          // https://github.com/mozilla/geckodriver/issues/776
+          await this.getActions()
+            .move({ x: 0, y: 0 })
+            .perform();
+          if (element instanceof WebElementWrapper) {
+            return await this.getActions()
```

Comment:
Can we just return on the next line rather than changing the function signature away from `Promise<void>` please?